shell/browser/ui/inspectable_web_contents.cc

```diff
@@ -737,9 +742,10 @@ void InspectableWebContents::ShowItemInFolder(
     return;
 
   base::FilePath path = base::FilePath::FromUTF8Unsafe(file_system_path);
-
-  // Pass empty callback here; we can ignore errors
-  platform_util::OpenPath(path, platform_util::OpenCallback());
+  platform_util::OpenPath(
```

Comment:
Should we just call `ShowItemInFolder` here? The `OpenPath` will open the file, which I believe was a mistake.

Reply:
@zcbenz it's also what Chromium does, see linked code above